From ac4ad6ee532dd2709e4763816f233e13acbcf3ad Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:49:59 -0400
Subject: [fs] nfsd: minor nfsd4_replay_cache_entry cleanup

Message-id: <1403560314-32525-58-git-send-email-bfields@redhat.com>
Patchwork-id: 84496
O-Subject: [Patch RHEL7 057/172] nfsd4: minor nfsd4_replay_cache_entry cleanup
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

Maybe this is comment true, who cares?  Handle this like any other
error.

Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: 0da7b19cc3f9c7e762de02cb2152eaf20272a144
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/nfs4state.c b/fs/nfsd/nfs4state.c
index a08b58d..451638b 100644
--- a/fs/nfsd/nfs4state.c
+++ b/fs/nfsd/nfs4state.c
@@ -1605,9 +1605,8 @@ nfsd4_replay_cache_entry(struct nfsd4_compoundres *resp,
 
  dprintk("--> %s slot %p\n", __func__, slot);
 
- /* Either returns 0 or nfserr_retry_uncached */
  status = nfsd4_enc_sequence_replay(resp->rqstp->rq_argp, resp);
- if (status == nfserr_retry_uncached_rep)
+ if (status)
   return status;
 
  /* The sequence operation has been encoded, cstate->datap set. */
-- 
1.7.1